converter/include/k4SimDelphes/k4GenParticlesDelphesConverter.h

```diff
@@ -0,0 +1,68 @@
+#ifndef K4GENPARTICLESDELPHESCONVERTER_H
+#define K4GENPARTICLESDELPHESCONVERTER_H
+
+#include "edm4hep/MCParticleCollection.h"
+#include "classes/DelphesFactory.h"
+#include "classes/DelphesClasses.h"
+#include "TObjArray.h"
+
+
+
+// todo: move to converter and test
+
+namespace k4SimDelphes {
+
+class k4GenParticlesDelphesConverter {
```

Comment:
Can this be a free function? It doesn't seem to hold any state.

Reply:
yes, but I'll need to introduce some state for the workaround for the  MCRecoParticleAssociations.